modules/thunder_workflow/css/edit-form.css

```diff
@@ -0,0 +1,9 @@
+.form-actions div .form-item {
+  margin: 0;
+}
+
+.form-actions div {
```

Comment:
Selector is too greedy, limit to .gin-sticky, since there could be multiple form-actions.
Also only direct child div should receive styling:
`.gin-sticky .form-actions > div`